# StopTracking.srv
---
bool success
string message